quantum/action_tapping.c

```diff
@@ -377,13 +385,8 @@ bool process_tapping(keyrecord_t *keyp) {
             // clang-format on
             if (event.pressed) {
                 if (IS_TAPPING_RECORD(keyp)) {
-//#    ifndef TAPPING_FORCE_HOLD
-#    if !defined(TAPPING_FORCE_HOLD) || defined(TAPPING_FORCE_HOLD_PER_KEY)
-                    if (
-#        ifdef TAPPING_FORCE_HOLD_PER_KEY
-                        !get_tapping_force_hold(tapping_keycode, &tapping_key) &&
-#        endif
-                        !tapping_key.tap.interrupted && tapping_key.tap.count > 0) {
+#    if defined(QUICK_TAP_TERM) && QUICK_TAP_TERM > 0
```

Comment:
Unlike feature options that can be turned ON or OFF (like for example retro-tap), it doesn't make much sense to `undef` a term/time period. You could remove all those `ifdef` checks by defining the default value of `QUICK_TAP_TERM` in `quantum/action_tapping.h`, in [the same fashion as for the TAPPING_TERM](https://github.com/qmk/qmk_firmware/blob/4bac5f53d864a77a6f0fa8a2a046ed7748824ecc/quantum/action_tapping.h#L20..L23=)

Reply:
The advantage of maintaining check for `QUICK_TAP_TERM > 0` is code size reduction for users that use `0`, because AVR. I'll review firmware size savings again before deciding on this.

Reply:
From my tests, `#if 0 > 0` strips the code inside the if guards just fine.
```c
#define MY_TERM 0
#if MY_TERM > 0
    my_function();
#endif
```
`my_function()` doesn't appear in the produced binary for my AVR Dactyl Manuform.